BaseStation/MainWindow/MainWindow.xaml.cs

```diff
@@ -13,13 +15,25 @@ public partial class MainWindow : Window {
 		private SettingsFile SettingsFile = new SettingsFile(SETTINGS_PATH);
 		public Settings Settings { get => SettingsFile.Settings; }
 		public ObservableDictionary<string, MeasuredValue<double>> Properties { get; }
+        public Armature SetpointArm;
 
-		public MainWindow() {
-			Properties = new MockObservableMap();
-			InitializeComponent();
-			WindowState = WindowState.Maximized;
+		public MainWindow()
+        {
+            Properties = new MockObservableMap();
+            InitializeComponent();
+            WindowState = WindowState.Maximized;
 			DataContext = this;
-		}
+
+            double degToRad = Math.PI / 180;
+            ArmSideViewer.SetpointArmature = 
```

Comment:
What is the intention of changing the MainWindow, it looks like you are removing Map view stuff?